llms/mlx_lm/generate.py

```diff
@@ -188,6 +189,8 @@ def main():
     elif using_cache:
         tokenizer.chat_template = metadata["chat_template"]
 
+    prompt = codecs.decode(args.prompt, "unicode_escape")
+
```

Comment:
This is a random addition, but it lets you do things like `Hello\nworld` on the CLI and have the return character be parsed correctly.